katalogus/templates/partials/objects_to_scan.html

```diff
@@ -22,20 +22,6 @@ <h2>{{ form_title }}</h2>
         </form>
       </div>
     </div>
-
-    {% if 'show_all=on' in request.get_full_path %}
```

Comment:
This should not be removed.

Reply:
Fixed in commit: https://github.com/minvws/nl-kat-rocky/pull/111/commits/4258c854ad7c0d2f9c32199056bf43355e96e79f

Reply:
We dont need to ask this if there is a get request to ?show_all which will show the list of boefjes that dont meet the requirements, moving the warning after if statement is enough